tools/nnicmd/launcher.py

```diff
@@ -297,20 +296,29 @@ def launch_experiment(args, experiment_config, mode, experiment_id=None):
     
     #save experiment information
     experiment_config = Experiments()
-    experiment_config.add_experiment(experiment_id, args.port, start_time)
+    experiment_config.add_experiment(experiment_id, args.port, start_time, CONFIG_FILE_NAME)
 
     print_normal(EXPERIMENT_SUCCESS_INFO % (experiment_id, '   '.join(web_ui_url_list)))
 
 def resume_experiment(args):
     '''resume an experiment'''
-    nni_config = Config(args.port)
+    experiment_config = Experiments()
+    experiment_dict = experiment_config.get_all_experiments()
+    if experiment_dict.get(args.id) is None:
+        print_error('Id not exist!')
```

Comment:
How about 'Experiment ID {} does not exist.' to put sepefic exp id in error message.